Scripts/script-ReadPDFFile.yml

```diff
@@ -84,7 +99,8 @@ script: |-
           demisto.results({"Type" : entryTypes["note"],
                           "ContentsFormat" : formats["markdown"],
                           "Contents" : md,
-                          "HumanReadable": md
+                          "HumanReadable": md,
+                          "EntryContext": {"File(val.EntryID == obj.EntryID)" : pdfFile, "URL" : URLs}
```

Comment:
format the dict
https://github.com/demisto/content/blob/master/docs/code_conventions/README.MD